docs: add example and docs about using Python libraries in a project (#190)
vmware-labs/wasm-workers-server: docs/docs/languages/python.md

If you prefer, you can configure the environment variable value dynamically by following [these instructions](../features/environment-variables.md#inject-existing-environment-variables).

## Python libraries

> The Python ecosystem has a huge number of packages. Developers like you usually rely on other libraries to accomplish different goals. In this example, you will use the [Beautiful Soup](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#quick-start) to parse a HTML document and read its text. You have all the code available in the [examples/python-libs](https://github.com/vmware-labs/wasm-workers-server/tree/main/examples/python-libs) folder.

Reviewer: ```suggestion
The Python ecosystem has a huge number of packages. Developers like you usually rely on other libraries to accomplish different goals. In this example, you will use the [Beautiful Soup](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#quick-start) to parse an HTML document and read its text. You have all the code available in the [examples/python-libs](https://github.com/vmware-labs/wasm-workers-server/tree/main/examples/python-libs) folder.
```

To add a new Python library to your project, follow these steps:

> 1. First, create an `index.py` file with the content below. This worker reads a HTML document and return the text using the `bs4` library:

Reviewer: ```suggestion
1. First, create an `index.py` file with the content below. This worker reads an HTML document and returns the text using the `bs4` library:
```

    ```python title="./index.py"
    from bs4 import BeautifulSoup

    html_doc = """<!DOCTYPE html>
    <head>
        <title>Wasm Workers Server</title>
    </head>
    <body>
    <main>
        <p>This page was generated by a Python file running in WebAssembly.</p>
    </main>
    </body>
    """

    def worker(req):
        soup = BeautifulSoup(html_doc, 'html.parser')

        res = Response(soup.get_text(". ", True))
        res.headers["x-generated-by"] = "wasm-workers-server"

        return res
    ```

1. Install the Beautiful Soup library with `pip` and save it in a new `_libs` folder:

    ```plain
    pip3 install -t ./_libs beautifulsoup4
    ```

> 1. Create a `index.toml` file with the content below. Note the name of the TOML file must match the name of the worker (`index.py` and `index.toml`). The current configuration mounts the `_libs` folder and sets the `PYTHONPATH` environment variable to the mount path:

Reviewer: ```suggestion
1. Create an `index.toml` file with the content below. Note the name of the TOML file must match the name of the worker (`index.py` and `index.toml`). The current configuration mounts the `_libs` folder and sets the `PYTHONPATH` environment variable to the mount path:
```

    ```toml title="./index.toml"
    name = "libs"
    version = "1"

    [vars]
    PYTHONPATH = "/opt/python/libs"

    [[folders]]
    from = "./_libs"
    to = "/opt/python/libs"
    ```

1. If you didn't download the `wws` server yet, check our [Getting Started](../get-started/quickstart.md) guide. You also need to install the Python runtime with the command below:

    ```plain
    wws runtimes install python latest
    ```

1. Run your worker with `wws`:

    ```bash
    wws

    ⚙️  Preparing the project from: .
    ⚙️  Loading routes from: .
    ⏳ Loading workers from 1 routes...
    ✅ Workers loaded in 524.804167ms.
        - http://127.0.0.1:8080/
        => ./index.py
    🚀 Start serving requests at http://127.0.0.1:8080
    ```

1. Finally, open <http://127.0.0.1:8080> in your browser.

### Limitations

> Currently, Wasm Workers Server only supports pure Python libraries like Beautiful Soup. Libraries that requires to compile native extensions are not available yet.

Reviewer: ```suggestion
Currently, Wasm Workers Server only supports pure Python libraries like Beautiful Soup. Libraries that requires to compile native extensions are not supported yet.
```
